### PR TITLE
Fix Accessibility Issue on the theme button on the NavBar page

### LINKED ISSUE #67: Fix Accessibility Issue on the theme button on the NavBar page
### Description

Screenshot (133)

I would like to add an aria-label attribute to the `button`

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Yagazie-davidson**
Closes #67

### CODE REVIEW

File: components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ const Navbar = () => {
       {/* Nav Links */}
       <ul data-nav-toggle={navToggle} className="animation">
         <button
+          aria-label="theme toogle"
```
Comment (alisamadiii) [inline components/Navbar/Navbar.jsx]: There is a spelling mistake, Can we correct it?
```diff
- toogle
+ toggle
```

**Review by alisamadiii — CHANGES_REQUESTED** [review]
The Rest LGTM 🎉

File: components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Yagazie-davidson) [reply components/Navbar/Navbar.jsx]: Yeah, I would!

**Review by alisamadiii — APPROVED** [review]
LGTM 🎉

**Comment by alisamadiii** [comment]
Great Job,
Thanks for your contributions 🎉

**Pull request merged**